applications/callflow/src/module/cf_ring_group.erl

```diff
@@ -34,7 +34,10 @@ handle(Data, Call) ->
 -spec attempt_endpoints(wh_json:objects(), wh_json:object(), whapps_call:call()) -> 'ok'.
 attempt_endpoints(Endpoints, Data, Call) ->
     Timeout = wh_json:get_integer_value(<<"timeout">>, Data, ?DEFAULT_TIMEOUT_S),
-    Strategy = wh_json:get_binary_value(<<"strategy">>, Data, ?DIAL_METHOD_SIMUL),
+    Strategy = case wh_json:get_binary_value(<<"strategy">>, Data, ?DIAL_METHOD_SIMUL) of
```

Comment:
I think if you rebase to master, now that KAZOO-3942 is in there (with ring group schema), we can go back to just fetching the value. Otherwise, let's put this in a function.